glean-core/src/traits/counter.rs

```diff
@@ -45,9 +45,5 @@ pub trait Counter {
     /// # Returns
```

Comment:
All of these traits helpfully include doc comments referencing `ping_name` that should probably be removed.